orchestra/models/core/models.py

```diff
@@ -753,6 +753,10 @@ class Todo(TodoMixin, BaseModel):
         activity_log (str)
             A JSON blob that records the user actions
             with this todo
+        slug (str)
+            A unique identifier for each todo list item.
+            It is used to reference and retrieve specific
```

Comment:
It is used to refer and retrieve specific...

Reply:
to-do items. Line below. Or do you mean something else?

Reply:
It is used to reference -> It is used to refer

Reply:
oh sorry, didn't notice